src/main/java/frc/robot/Robot.java

```diff
@@ -33,7 +33,7 @@ public void robotInit() {
     m_robotContainer = new RobotContainer();
   }
 
-  /**
+  /*
```

Comment:
When describing a function you should use /** so that the ide knows to show the comment when referencing that function.